src/Lean/Elab/Tactic/Ext.lean

```diff
@@ -249,6 +246,7 @@ def applyExtTheoremAt (goal : MVarId) : MetaM (List MVarId) := goal.withContext
 @[builtin_tactic applyExtTheorem] def evalApplyExtTheorem : Tactic := fun _ => do
   liftMetaTactic applyExtTheoremAt
 
+open TSyntax.Compat in
```

Comment:
I don't understand why un-`public section`ioning this affects the `TSyntax` type checking magic.